app/script/connect/ConnectRepository.coffee

```diff
@@ -121,33 +120,33 @@ class z.connect.ConnectRepository
   @return [z.connect.PhoneBook] Encoded phone book data
   ###
   _parse_macos_contacts: ->
-    return if not window.zAddressBook
-
-    address_book = window.zAddressBook()
-    phone_book = new z.connect.PhoneBook @properties_repository.self()
+    return new Promise (resolve, reject) =>
+      return resolve undefined if not window.zAddressBook
+      address_book = window.zAddressBook
```

Comment:
right.. shall we rename the `zAddressBook` to `wAddressBook` maybe? That should work.. and also maybe removing the option to import from AddressBook if that module is missing..